src/GNDStk/gnds-utility.hpp

```diff
@@ -0,0 +1,231 @@
+
+// -----------------------------------------------------------------------------
+// Macros, variables, enums
+// -----------------------------------------------------------------------------
+
+// printval
+// printvar
+// Are the same, so we don't need to remember the exact terminology
+#define printval(var) std::cout << #var ": [" << var << "]" << std::endl
```

Comment:
Is this a convenience while developing?

Reply:
Yes, precisely. Much like the Boost name-demangle stuff that I'd put in as conditionally included material.